sop/modules/OSD/partials/troubleshoot_ceph_err.adoc

```diff
@@ -14,6 +14,20 @@
 The first two status commands provide the overall cluster health. The normal state for cluster operations is HEALTH_OK, but will still function when the state is in a HEALTH_WARN state. If you are in a WARN state, then the cluster is in a condition that it may enter the HEALTH_ERROR state at which point all disk I/O operations are halted. If a HEALTH_WARN state is observed, then one should take action to prevent the cluster from halting when it enters the HEALTH_ERROR state.
 
 Problem 1::
+Alert Manager shows that the OSD is full
```

Comment:
```suggestion
Ceph status shows that the OSD is full
```

Reply:
Would ceph status show that osd is full? last we checked it did not right?

Reply:
Yeah, it will show OSDs full when it crosses 85%